examples/rsp-cra-18/src/BodyContent.tsx

```diff
@@ -1,5 +1,5 @@
-import {useState, FormEvent, Key, useRef} from "react";
-import {Item, TabList, TabPanels, Tabs} from '@adobe/react-spectrum'
+import {useState, FormEvent, useRef} from "react";
+import {Key, Item, TabList, TabPanels, Tabs} from '@adobe/react-spectrum'
```

Comment:
this is fine, we could also delete the usage of it like we have in webpack 4 app